Doc/tutorial/introduction.rst

```diff
@@ -11,6 +11,13 @@ with a prompt are output from the interpreter. Note that a secondary prompt on a
 line by itself in an example means you must type a blank line; this is used to
 end a multi-line command.
 
+.. only:: html
+
+   You can toggle the display of prompts and output by clicking the ``>>>``
+   link in the upper-right corner of an example box.  If you hide the prompts
```

Comment:
Could another word than `link` be used?  It’s not a real link as it doesn’t go to another page, can’t be right-clicked for context menu, etc.

Reply:
@merwok That's a good point. What do you think about the word `button`? I also thought about `prompt`, though most prompts on the page are not clickable, just these special ones. Or perhaps just this shorter text instead:
```
   You can toggle the display of prompts and output by clicking on ``>>>``
   in the upper-right corner of an example box.  If you hide the prompts
```

Reply:
Great, I updated the phrasing in the latest commit.